app/controllers/application_controller.rb

```diff
@@ -120,6 +122,10 @@ def authenticate_user!
     end
   end
 
+  def set_default_view_variables
+    @adv_search = Query::Proposal::FieldedSearch.new({})
```

Comment:
I agree with @amberwreed. Let's keep this on the My Requests + My Dashboard and remove elsewhere. I could definitely see this being globally available if supported by a design that has a global nav, but it doesn't have that treatement atm.